source/server/http/admin.cc

```diff
@@ -412,6 +449,8 @@ AdminImpl::AdminImpl(const std::string& access_log_path, const std::string& prof
           {"/server_info", "print server version/status information",
            MAKE_ADMIN_HANDLER(handlerServerInfo), false},
           {"/stats", "print server stats", MAKE_ADMIN_HANDLER(handlerStats), false},
+          {"/metrics", "print server stats in prometheus' format",
```

Comment:
I can't remember whether we've had this conversation; would it make sense to have all the different stats format under `/stats`, so similar to JSON we would have `/stats?format=prometheus`? Just raising the question for discussion, since I can anticipate further stats format, are we going to look for more synonyms for "stats" when that happens? :)

Reply:
This is the way it used to be, but we changed it back to `/metrics` because that is the defualt Prometheus path. In thinking about it, I agree with @htuch that we should switch it back to `/stats?format=prometheus`. We are not a Prometheus only operation so this is more consistent and it will be well documented.

Reply:
That's fine. I can change it back.